ci-operator/config/openshift/cluster-api-provider-libvirt/master.json

```diff
@@ -29,7 +29,7 @@
     {
       "as": "unit",
       "from": "src",
-      "commands": "make test"
+      "commands": "yum install -y libvirt-devel && make test"
```

Comment:
the issue with deps is likely going to happen with all the command not only unit, so might wanna set a more generic approach moving this to a script or similar

Reply:
Maybe into Makefile?

Reply:
yeah, just something to reuse on CI, but not to run it by default on dev as there's different envs to install stuff so that can be left to the user for now